tests/testcpp.h

```diff
@@ -96,6 +96,27 @@ namespace Test {
                               (int)randomized);
   }
 
+  /**
+  * @brief Create a topic using the Admin API
+  */
+  static RD_UNUSED void create_topic (RdKafka::Handle *use_handle, const char *topicname,
+                                      int partition_cnt, int replication_factor) {
+    rd_kafka_t *use_rk = NULL;
+    if (use_handle != NULL)
+      use_rk = use_handle->c_ptr();
+    test_create_topic(use_rk, topicname, partition_cnt, replication_factor);
+  }
+
+  /**
+  * @brief Delete a topic using the Admin API
```

Comment:
indent

Reply:
"using the Admin API" is not true, it will fall back to kafka-topics if not available.